### PR TITLE
css-two-column-layout - Ray Le

### PR DESCRIPTION
**Pull request by lol-ray**
image

### CODE REVIEW

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Very nice work Ray! the only update I see is to make sure to add a class to the CSS for your `col-half` and set it to 50% width. 

Great work!

**Comment by lol-ray** [comment]
image

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  text-align: center;
+  margin-top: 20px;
+}
+
+.col-full > p {
+  padding: 0 170px;
+}
+
+.col-half {
+  width: 100%;
```
Comment (Zhiwutian) [inline css-two-column-layout/styles.css:47]: make sure its `50%` :)

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny tiny update!

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (lol-ray) [reply css-two-column-layout/styles.css:47]: OH oops

**Comment by lol-ray** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼 Very Nice! Congrats!

**Pull request merged**